docs/src/components/landing-page/FeatureBanner.js

```diff
@@ -0,0 +1,102 @@
+ *                                                                                *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy   *
+ * of this software and associated documentation files (the "Software"), to deal  *
+ * in the Software without restriction, including without limitation the rights   *
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell      *
+ * copies of the Software, and to permit persons to whom the Software is          *
+ * furnished to do so, subject to the following conditions:                       *
+ *                                                                                *
+ * The above copyright notice and this permission notice shall be included in all *
+ * copies or substantial portions of the Software.                                *
+ *                                                                                *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR     *
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,       *
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE    *
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER         *
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,  *
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE  *
+ * SOFTWARE.                                                                      *
+ **********************************************************************************/
+import DashboardIconSvg from '../../../public/dashboard-icon.svg'
+import TimerIconSvg from '../../../public/timer-icon.svg'
+import AlarmIconSvg from '../../../public/alarm-icon.svg'
+import ListIconSvg from '../../../public/list-icon.svg'
+import RecycleIconSvg from '../../../public/recycle-icon.svg'
+import WhatsAppIconSvg from '../../../public/whatsapp-icon.svg'
+
+export default function FeatureBanner() {
+  return (
+    <div className="relative flex flex-col bg-white py-32 z-10 transform -skew-y-6 -mt-20">
+      <div className="w-1/4 h-6 absolute top-0 right-0 bg-gradient-to-r from-purple-monika to-aqua-monika" />
+      <div className="w-1/4 h-6 absolute bottom-0 left-0 bg-gradient-to-r from-purple-monika to-aqua-monika" />
+      <div className="m-auto mx-1/6 flex flex-col z-10 transform skew-y-6">
+        <h2 className="mr-auto bg-gradient-to-r from-purple-monika to-aqua-monika bg-clip-text text-transparent font-bold text-xl">
+          Why Monika
+        </h2>
+        <h2 className="text-3xl">
+          Free and Open Source Synthetic
+          <br />
+          Monitoring Tool
+        </h2>
+        <div className="grid grid-cols-1 sm:grid-cols-2 lg:grid-cols-2 xl:grid-cols-3 gap-2 grid-flow-row mt-4">
+          <div>
+            <img src={DashboardIconSvg} />
```

Comment:
Please add `className="max-h-20"` to all feature icons so that they are all properly aligned.

Before:
<img width="1015" alt="Screenshot 2021-06-15 at 20 23 51-1" src="https://user-images.githubusercontent.com/311343/122105000-93414f80-ce18-11eb-94f3-02d7ae37f2eb.png">

After:
<img width="985" alt="Screenshot 2021-06-15 at 20 29 32-2" src="https://user-images.githubusercontent.com/311343/122105032-9b998a80-ce18-11eb-9e18-48c721cd6655.png">